src/improvements/tasks/StateOverrideManager.sol

```diff
@@ -87,27 +89,43 @@ abstract contract StateOverrideManager {
     }
 
     /// @notice Create default state override for the parent multisig.
-    function _createDefaultTenderlyOverride(address parentMultisig, uint256 nonce)
+    function _parentMultisigTenderlyOverride(address parentMultisig, uint256 nonce)
         private
-        view
+        pure
         returns (Simulation.StateOverride memory)
     {
         Simulation.StateOverride memory defaultOverride;
         defaultOverride.contractAddress = parentMultisig;
+        defaultOverride = _overrideMultisigThresholdAndNonce(defaultOverride, 1, nonce);
+        return defaultOverride;
```

Comment:
nit, can do the same in `_childMultisigTenderlyOverride` too
```suggestion
        returns (Simulation.StateOverride memory defaultOverride)
    {
        defaultOverride.contractAddress = parentMultisig;
        defaultOverride = _overrideMultisigThresholdAndNonce(defaultOverride, 1, nonce);
```

Reply:
Implemented here: https://github.com/ethereum-optimism/superchain-ops/pull/771/commits/2979b3c74d89bba4ca8fb0c6774bb122400ecad3